pkg/ddevapp/global_dotddev_assets/commands/web/create-sitepackage

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+#ddev-generated
+
+## Description: Create a default TYPO3 composer-style sitepackage
+## Usage: create-sitepackage
+## Example: "ddev create-sitepackage"
+## ProjectTypes: typo3
+## ExecRaw: true
+## MutagenSync: true
+set -eu -o pipefail
+
+
+cd ${DDEV_COMPOSER_ROOT:-/var/www/html}
+
+# Create the required data.json
+cat << 'EOF' > /tmp/data.json
+{
+  "base_package": "site_package_tutorial",
+  "typo3_version": 13.4,
+  "title": "My Site Package",
+  "description": "Site package to follow the tutorial. ",
+  "repository_url": "https://github.com/TYPO3-Documentation/TYPO3CMS-Tutorial-SitePackage",
+  "author": {
+    "name": "J. Doe",
+    "email": "j.doe@example.org",
+    "company": "My Vendor",
+    "homepage": "https://docs.typo3.org/permalink/t3sitepackage:start"
+  }
+}
+EOF
+
+curl -X 'POST' \
+  'https://get.typo3.org/api/v1/sitepackage/' \
+  -s \
+  --fail \
+  -H 'accept: application/zip' \
+  -H 'Content-Type: application/json' \
+  -d @"/tmp/data.json" --output /tmp/my_site_package.zip
+
+composer require my-vendor/my-site-package:@dev
+
+rm -rf packages/my_site_package/*
```

Comment:
I think we should not remove the site package if one exist unless some kind of `--force` parameter was set

Reply:
Your wish is my command. Updated.